backend_starter_app/tests.py

```diff
@@ -0,0 +1,10 @@
+from django.test import TestCase
```

Comment:
I have not looked into Django test framework support - is this based on Pytest or UnitTest runner ?

Reply:
django.test is based on unittest

Reply:
Okay. I vouch to leave it as is for now - maybe think of what to recommend long term.

The reason is that the E2E tests ran by the QE department mainly use Pytest as the test framework, so there's a chance that a big user base is already familiar with Pytest. 

We should examine other Django projects in the platform and consider whether to keep Unittest or add the Pytest runner instead. 

Let's tackle this in a separate task. I'll create a card for it.